internal/manifests/collector/annotations.go

```diff
@@ -66,7 +68,11 @@ func PodAnnotations(instance v1alpha1.OpenTelemetryCollector) map[string]string
 	return podAnnotations
 }
 
-func getConfigMapSHA(config string) string {
-	h := sha256.Sum256([]byte(config))
+func getConfigMapSHA(config v1alpha2.Config) string {
```

Comment:
How does it impact the upgrade? It seems like the SHA is changing with the new version

Reply:
Thats actually a good point. I assume it does and the operator will update all pods? How does it work during an operator upgrade increasing the version from e.g. `0.88.0` to `0.89.0`? The config checksum would be the same, but the container images would be different?

Reply:
yeah i think the SHA changing is fine, it would result in a new replica set iirc. Same as what would happen on a version upgrade. Acceptable IMO

Reply:
If this is a problem, it should be detected by the E2E tests